flake8_annotations/__init__.py

```diff
@@ -244,3 +261,17 @@ def visit_ClassDef(self, node: ast.ClassDef) -> None:
                 for method_node in method_nodes
             ]
         )
+
+    @classmethod
+    def parse_file(cls, src_filepath: Path) -> "FunctionVisitor":  # Need to quote for 3.6 compat
```

Comment:
Is this used anywhere in this package or only for testing? If the latter, I think this should be in `testing/__init__.py`.

Reply:
I don't mind these where they are. A few plugins (e.g. [bugbear](https://github.com/PyCQA/flake8-bugbear/blob/master/bugbear.py#L56)) have this method as a fallback as well. If I could figure out what scenario causes the lines/ast to not be provided by flake8 I'd add it as well, but until I can figure that out I don't know how to test it.